config/autoconfigbackup/autoconfigbackup.inc

```diff
@@ -1,80 +1,100 @@
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+	AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+	AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+	OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+	SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+	INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+	ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+	POSSIBILITY OF SUCH DAMAGE.
 */
-
 require_once("filter.inc");
 require_once("notices.inc");
 
-$pf_version=substr(trim(file_get_contents("/etc/version")),0,3);
-if ($pf_version < 2.0)
+$pf_version = substr(trim(file_get_contents("/etc/version")), 0, 3);
+if ($pf_version < 2.0) {
 	require_once("crypt_acb.php");
+}
 
 // Plugin moved to save only
-if(file_exists("/usr/local/pkg/parse_config/parse_config_upload.inc"))
-	unlink("/usr/local/pkg/parse_config/parse_config_upload.inc");
-if(file_exists("/usr/local/pkg/parse_config/parse_config_upload.php"))
-	unlink("/usr/local/pkg/parse_config/parse_config_upload.php");
+unlink_if_exists("/usr/local/pkg/parse_config/parse_config_upload.inc");
+unlink_if_exists("/usr/local/pkg/parse_config/parse_config_upload.php");
 
-/* ensures patches match */
-function custom_php_validation_command($post, &$input_errors) {
+// Ensures patches match
+function acb_custom_php_validation_command($post, &$input_errors) {
 	global $_POST, $savemsg, $config;
 
-	if(!$post['username'])
+	if (!$post['username']) {
 		$input_errors[] = "Username is required.";
+	}
 
-	if(!$post['password'] or !$post['passwordagain'])
+	if (!$post['password'] or !$post['passwordagain']) {
 		$input_errors[] = "The subscription password is required.";
+	}
 
-	if(!$post['crypto_password'] or !$post['crypto_password2'])
+	if (!$post['crypto_password'] or !$post['crypto_password2']) {
 		$input_errors[] = "The encryption password is required.";
+	}
 
-	if($post['password'] <> $post['passwordagain']) 
+	if ($post['password'] <> $post['passwordagain']) {
 		$input_errors[] = "Sorry, the entered portal.pfsense.org passwords do not match.";
+	}
 
-	if($post['crypto_password'] <> $post['crypto_password2']) 
+	if ($post['crypto_password'] <> $post['crypto_password2']) {
 		$input_errors[] = "Sorry, the entered encryption passwords do not match.";
+	}
 
-	if($post['testconnection']) {
+	if ($post['testconnection']) {
 		$status = test_connection($post);
-		if($status) 
+		if ($status) {
 			$savemsg = "Connection to portal.pfsense.org was tested with no errors.";
+		}
 	}
-	
+
 	// We do not need to store this value.
 	unset($_POST['testconnection']);
 }
 
+function acb_custom_php_resync_config_command() {
+	conf_mount_rw();
+	unlink_if_exists("/cf/conf/lastpfSbackup.txt");
+	conf_mount_ro();
```

Comment:
Would be even better if unlink_if_exists() handled this on its own... :stuck_out_tongue: